android/app/src/main/res/values/strings.xml

```diff
@@ -158,6 +158,8 @@
     <string name="fmt_floor">"Floor: "</string>
     <string name="noBookmark">No bookmarked session yet !!</string>
 
+    <string name="transittype">transit</string>
```

Comment:
Please use more descriptive string name and value. Choose such a value that it represents the transition and does not clash with future ones :smile: